### DISCUSSION TITLE
Question about Hyperactive usage

### DISCUSSION THREAD

**Discussion by SimonBlanke** [wkpark/sd-webui-model-mixer#128] [General]
Hello there!

A very interesting project you have going here! :-)
I am curious about [this line](https://github.com/wkpark/sd-webui-model-mixer/blob/master/sd_modelmixer/hyper.py#L537C15-L537C23) in your source-code. After instantiating the `Hyperactive`-class the process-dictionary `opt_pros` (terrible variable name...) should be empty. Why the attempt to remove the `0`-key? If this is a bug on my side I would like to correct it.

Also: Cool to see `CustomOptimizationStrategy` in action here!

**Comment by wkpark** [comment]
https://github.com/Xerxemi/auto-MBW-rt/blob/2e8695f47332f5d76d8b8b3d1bf6f9ef5e90b297/scripts/main/mbw.py#L552

I didn't dig into further but that line was from the original source code.
